emsdk

```diff
@@ -487,6 +487,10 @@ def download_file(url, dstpath, download_even_if_exists=False, filename_prefix =
     print("Error downloading URL '" + url + "': " + str(e))
     rmfile(file_name)
     return None
+  except KeyboardInterrupt as e:
+    print("Received Interrupt, exiting")
+    rmfile(file_name)
+    exit(0)
```

Comment:
`exit(1)` would be better here, or otherwise if this was chained to scripts, it might continue executing the next script believing `emsdk` run had succeeded?

Reply:
Makes sense, the exit code the way it is now is 1.